ContactLog/People.js

```diff
@@ -111,6 +157,14 @@ class People extends Component {
         <TouchableOpacity onPress={this.openModal} style={styles.add_button}>
           <CustomIcon name={'add24'} color={'white'} size={20} />
         </TouchableOpacity>
+        {
+          !this.state.contactAccessIsAllowed && 
```

Comment:
ideally we don't have this button and check the user's permission when clicking on the plus icon.

A couple issues with the link 1) it doesn't match the design, the color is not accessible. 2) the alert won't take user to the OS settings. 

How about we remove this button from this PR and handling "enable access" separately?